tools/prologs-epilogs/receive-data-path-manager-mega

```diff
@@ -58,6 +58,14 @@ if [[ ${SLURM_SCRIPT_CONTEXT} == "prolog_slurmd" ]]; then
 		${NCCL_PLUGIN_IMAGE} \
 		install
 
+	# Modify NCCL env vars for Debian 12
```

Comment:
- Is this "profile.sh" file created by the NCCL_PLUGIN_IMAGE container above?
- i.e. appending to it is appropriate?
- we could probably add some magic here to detect Debian 12 in the OS so that expansion to other distributions has an obvious path forward.

Reply:
* Yes, written by NCCL_PLUGIN_IMAGE above so appending is appropriate. 
* I added some basic detection for Debian 12 similar to https://github.com/GoogleCloudPlatform/cluster-toolkit/blob/main/modules/file-system/parallelstore/scripts/install-daos-client.sh.